tethys_cli/proxyapps_commands.py

```diff
@@ -0,0 +1,219 @@
+from tethys_cli.cli_colors import write_error, write_success, write_info, write_warning
+
+from tethys_cli.cli_helpers import setup_django
+
+from django.db import IntegrityError
+
+
+def add_proxyapps_parser(subparsers):
+    # Setup list command
+    proxyapps_parser = subparsers.add_parser(
+        "proxyapp", help="Add proxy apps and list proxy apps into the Tethys Platform"
+    )
+    proxyapps_subparsers = proxyapps_parser.add_subparsers(title="Options")
+
+    proxyapps_list_parser = proxyapps_subparsers.add_parser(
+        "list", help="list available proxy apps in the current tethys installation"
+    )
+    proxyapps_list_parser.add_argument(
+        "-v",
+        "--verbose",
+        help="List all the attributes of each available proxy app",
+        action="store_true",
+    )
+
+    proxyapps_list_parser.set_defaults(func=list_proxyapps)
+
+    proxyapps_add_parser = proxyapps_subparsers.add_parser(
+        "add",
+        help="Add a new proxy app.",
+    )
+    proxyapps_add_parser.add_argument(
+        "name",
+        help='The proxy app name (e.g.: "My App").',
+    )
+    proxyapps_add_parser.add_argument(
+        "endpoint",
+        help='The proxy app endpoint (e.g.: "https://myproxyapp.com").',
+    )
+    proxyapps_add_parser.add_argument(
+        "description",
+        help='The proxy app description (e.g.: "The following proxy app is a proxy app").',
+        nargs="?",
+        default="",
+    )
+    proxyapps_add_parser.add_argument(
+        "logo_url",
+        help='A URL to a logo image to use for the proxy app tile (e.g.: "https://the-logo-of-myproxy-app.png").',
+        nargs="?",
+        default="",
+    )
+
+    proxyapps_add_parser.add_argument(
+        "tags",
+        help='The proxy app tags separated by commas (e.g.: "tag1, tag2, tag3").',
```

Comment:
I'm not sure if that is the correct syntax for tags. I think it would be something more like `tag1,"tag2","tag three"` (No spaces after commas and optional quotes to allow for tags with spaces). Please test to verify.